flamingo/plugins/feeds.py

```diff
@@ -50,6 +60,9 @@ def pre_build(self, context):
                     else:
                         fe_updated = ''
 
+                    if feed_config['type'] == 'podcast':
+                        fe_podcast_enclosure = i['podcast_mp3_url'] or ''
```

Comment:
I would prefer something format-agnostic here instead of hardcoding `_mp3_url` in the header name, as that would mean possibly adding `_ogg_url`, `_opus_url`, `_aac_url` … later on.
Could we have a `podcast_url` that is `audio/mpeg` by default but can also take a dict of mime-type/url pairs?

Simple:

    # mp3 by default
    podcast_url: http://some.example/episodes/10.mp3

Flexible:

    # mime/url pairs are also possible
    podcast_url:
        audio/mpeg: http://some.example/episodes/10.mp3
        audio/ogg: http://some.example/episodes/10.opus
        audio/flac: http://some.example/episodes/10.flac
        audio/ogg: http://some.example/episodes/10.ogg

This is however only really relevant if the RSS podcast spec actually supports offering multiple formats to the client and let them choose or if we generate multiple format specific feeds.

Reply:
> This is however only really relevant if the RSS podcast spec actually supports offering multiple formats to the client and let them choose or if we generate multiple format specific feeds.

The specification seems a little vague about multiple enclosures. For example here:
https://podcasters.apple.com/support/823-podcast-requirements
But it seems internet-consensus that every item can only have one _enclosure_.

But: We need to provide a length (in bytes) for our file anyway. So I suggest:

```
podcast:
  url: https://some.example/file.mp3
  size: 123456789
  type: audio/mpeg
```
Where `type` is optional and defaults to `audio/mpeg` if not set.

I have pushed an update.